__tests__/upload.js

```diff
@@ -282,3 +285,20 @@ test('upload a ZIP file', async () => {
 
   expect(base64PNG).toMatchImageSnapshot(imageSnapshotOptions);
 });
+
+test('upload a .txt (plain) file', async () => {
```

Comment:
Do we have a test that, the bot is sending an activity with `attachment.content`?

This is because, the attachment that the user send/upload, will also be a file. Only the bot sending the text attachment may handle as preview or download link.

```json
{
  "attachments": [
    {
      "content": "Hello, World!",
      "contentType": "text/plain"
    }
  ],
  "type": "message"
}